src/app/form/(formSteps)/components/LoadingSpinner.tsx

```diff
@@ -0,0 +1,22 @@
+"use client";
+import loadingIcon from "public/loading.gif";
+
+export const LoadingSpinner = ({
+  loading,
+  children,
+}: {
+  loading: boolean;
+  children: React.ReactNode;
+}) => {
+  return loading ? (
+    <div style={{ textAlign: "center" }} aria-live="polite">
```

Comment:
[dust] should this have a `role="status"` here? Which will have an implicit `aria-live="polite"`.
```suggestion
    <div style={{ textAlign: "center" }} aria-live="polite" role="status">
```
Source: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/status_role